### DISCUSSION TITLE
Question: Can the release procedure provide a "dist" version of the JS?

### DISCUSSION THREAD

**Discussion by njt1982** [InboxSDK/InboxSDK#924] [General]
It would be nice for there to be a bundled pre-build JS asset on the release, to save having to download the release and built yourself or go via the faff of setting up webpack?

**Comment by officialsuyogdixit** [comment]
It is already there at https://unpkg.com/@inboxsdk/core@latest/inboxsdk.js, but the problem is that there is no documentation for it MV3. Complete documentation is built for MV2. There is no way to publish MV2 code from now.
